code/stable/gamephysics/SRS/HTML/GamePhysics_SRS.html

```diff
@@ -607,7 +607,7 @@ <h2>Abbreviations and Acronyms</h2>
       <div class="section">
         <h1>Introduction</h1>
         <p class="paragraph">
-          Due to the rising cost of developing video games, developers are looking for ways to save time and money for their projects. Using an open source physics library that is reliable and free will cut down development costs and lead to better quality products.
+          Due to the rising cost of developing video games, developers are looking for ways to save time and money for their projects. Using an open source physics library that is reliable and free will cut down development costs and lead to better quality products. The document describes the program  based on the original, manually created version of <a href="https://github.com/smiths/caseStudies/tree/main/CaseStudies/gamephys">GamePhysics</a>.
```

Comment:
The link is incorrect.  main should be master.  That is the link should be: https://github.com/smiths/caseStudies/tree/master/CaseStudies/gamephys 
The same comment applies for all other instances of this url.